x-pack/plugin/ccr/qa/security/src/test/java/org/elasticsearch/xpack/ccr/FollowIndexSecurityIT.java

```diff
@@ -202,7 +202,7 @@ public void testForgetFollower() throws IOException {
 
             assertOK(client().performRequest(new Request("POST", "/" + forgetFollower + "/_ccr/pause_follow")));
 
-            try (RestClient leaderClient = buildLeaderClient(restClientSettings())) {
```

Comment:
This change is unexpected to me. Can you explain why it's needed here?

Reply:
Removing retention leases requires the admin role.